src/config.py

```diff
@@ -0,0 +1,31 @@
+from os import path
+import os
+
+# Getting the parent directory of this file. That will function as the project home.
+PROJECT_HOME = path.dirname(path.dirname(path.abspath(__file__)))
+
+# App config
+APP_NAME = "runapp"
+DEBUG = True
+
+# Logging
+LOGGING_CONFIG = path.join(PROJECT_HOME, 'config', 'logging.conf')
+
+# AWS S3 & RDS
+S3_CONFIG = path.join(PROJECT_HOME, 'config', 'aws_s3.conf')
+RDS_CONFIG = path.join(PROJECT_HOME, 'config', 'aws_rds.conf')
+
+# Database connection config
+DATABASE_PATH = path.join(PROJECT_HOME, 'data', 'sample','cases.db')
+SQLALCHEMY_DATABASE_URI = 'sqlite:////{}'.format(DATABASE_PATH)
+SQLALCHEMY_TRACK_MODIFICATIONS = True
+SQLALCHEMY_ECHO = False  # If true, SQL for queries made will be printed
+
+# Download data url 
+BASE_URL = 'https://raw.githubusercontent.com/CSSEGISandData/COVID-19/master/csse_covid_19_data/csse_covid_19_time_series'
+FILE_NAME = 'time_series_covid19_confirmed_global.csv'
+OUTPUT_PATH = os.path.join(PROJECT_HOME, 'data', 'sample', FILE_NAME)
+
+# # Acquire and process config
+# MAX_RECORDS_READ = 100
+# SENTIMENT_RAW_LOCATION = path.join(PROJECT_HOME,'data/tweet_sentiment.json')
```

Comment:
Comments/codes not used should be deleted.

Reply:
The unused code are deleted.